bin/server.dart

```diff
@@ -1,11 +1,42 @@
 import 'dart:io';
 import 'dart:isolate';
 
+import 'package:dotenv/dotenv.dart';
+import 'package:logging/logging.dart';
 import 'package:platform_decoder/handler/general.dart';
 import 'package:shelf_plus/shelf_plus.dart';
 
-void main() {
-  const numberOfIsolates = 8;
+int numberOfIsolates = 8;
+int defaultBindPort = 8090;
+
+final logger = Logger('Decoder');
+
+void loadConfig() {
+  var env = DotEnv(includePlatformEnvironment: true)..load();
+
+  numberOfIsolates =
+      int.tryParse(env.getOrElse('NUMBER_OF_ISOLATES', () => '8')) ?? 8;
+  defaultBindPort =
+      int.tryParse(env.getOrElse('DEFAULT_BIND_PORT', () => '8090')) ?? 8090;
+  int specPerIsolate =
+      int.tryParse(env.getOrElse('SPEC_PER_ISOLATE', () => '4')) ?? 4;
+  int specExpireDuration =
+      int.tryParse(env.getOrElse('SPEC_EXPIRE_DURATION', () => '300')) ?? 300;
+
+  logger.info('Starting Platform Decoder v2.1.0');
```

Comment:
Could we get it from the pubspec?

Reply:
Not without adding a package and using async, lots of overhead for no benefit